### DISCUSSION TITLE
Docker 'testing-alpine' is using server '1.30.5-d04b94b7' when 1.31.0 is out

### DISCUSSION THREAD

**Discussion by ccben87** [dani-garcia/vaultwarden#4718] [General]
Hi, 

I am consistently running with testing-alpine docker image and have noticed that the server version is now behind the 'latest' docker image. Could you please update the 'testing-alpine' docker image to use the latest server version. Thanks.

**Comment by BlackDex** [comment]
The testing images are using the latest commit hash as far as i know.
The version number will always be behind unless there has been a new merge/push after a release tag has been created.

For the `testing` tagged images the commit hash is the most important verification.

Also, if you check the `/admin/diagnostics` page you see it stats it is ok.
![image](https://github.com/dani-garcia/vaultwarden/assets/610450/0ba6408d-c64b-4529-baba-e25611611e6d)

**Comment by ccben87** [comment]
Yes, it does state 'Ok', but look at your own screenshot (same as for me and why I made this post) for where it states Server Installed is '1.30.5-d04b94b7' and Server Latest is '1.31.0-d04b94b7'

**Reply by BlackDex** [reply]
Yes, i know, read my explanation ;)

**Reply by ccben87** [reply]
Oh, sorry, thanks for the explanation.
